src/kartikeyDJune20/MeetingRoomUsingEnum.java

```diff
@@ -0,0 +1,101 @@
+package kartikeyDJune20;
+
+
+public class MeetingRoomUsingEnum {
+	enum meeetingRooms{
+		
+		NALANDA, TAKSHILA, AGRAFORT, PRATAPGARDH;
+	}
+	static boolean nalandaBooked = false;
+	static boolean takshilaBooked = false;
+	static boolean agraFortBooked = false;
+	static boolean pratapGardhBooked = false;
+	static int nalandaBookingTime, takshilaBookingTime, agrafortBookingTime, pratapGrardhBookingTime;
+
+	void bookMeetingRoom() {
+		bookmeetigRoom(meeetingRooms.NALANDA);
+	}
+
+	void bookmeetigRoom(meeetingRooms roomName, int time) {
+		switch (roomName) {
+		case NALANDA:
+			if (nalandaBooked == false) {
+				nalandaBooked = true;
+				nalandaBookingTime = time;
+			} else
+				System.out.println(roomName+" room you want is already booked for "+nalandaBookingTime+" hrs");
+			break;
+		case TAKSHILA:
+			if (takshilaBooked == false) {
+				takshilaBooked = true;
+				takshilaBookingTime = time;
+			} else
+				System.out.println(roomName+" room you want is already booked for "+takshilaBookingTime+" hrs");
+			break;
+		case AGRAFORT:
+			if (agraFortBooked == false) {
+				agraFortBooked = true;
+				agrafortBookingTime = time;
+			} else
+				System.out.println(roomName+" room you want is already booked for "+agrafortBookingTime+" hrs");
+			break;
+		case PRATAPGARDH:
+			if (pratapGardhBooked == false) {
+				pratapGardhBooked = true;
+				pratapGrardhBookingTime = time;
+			} else
+				System.out.println(roomName+" room you want is already booked for "+pratapGrardhBookingTime+" hrs");
+			break;
+		default:
```

Comment:
Please ignore this as i forget it to comment .. we are using enum class so there is no use of it..